frontend/src/tests/HomePage.spec.js

```diff
@@ -0,0 +1,34 @@
+import Enzyme, { configure } from "enzyme";
+import React from 'react';
+import Adapter from "enzyme-adapter-react-16";
+import fetch from "node-fetch";
+import { mount, shallow } from 'enzyme';
+import LandingPage from "../homePage/LandingPage"
+import NavBar from "../homePage/NavBar"
+
+Enzyme.configure({ adapter: new Adapter() });
+global.fetch = fetch;
+
+describe('Home Page component testing', () => {
+  test("Landing page loads button with alternating text", () => {
+    const wrapper = mount(<LandingPage loginStatus={true} />);
+    expect(wrapper.find('.MyButton').text()).toEqual(" My Folders ");
```

Comment:
It's better to replace these tests with snapshot testing. They are easier to read and maintain (to update tests, you just have to press a button).